types/robust-segment-intersect/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "esModuleInterop":true,
```

Comment:
Same here; this line should be removed to make sure that old projects work with the types.